### ISSUE TITLE
Using folder as path for project import produces a crash

### ISSUE THREAD

**Issue by Mai-Lapyst** [eumario/godot-manager#54]
**Describe the bug**
When trying to import a project from disk, and choosing the folder as path instead of the `project.godot` file, the application crashes.

**To Reproduce**
1. Go to the project manager
2. Click on "Import Project"
3. Put as path the folder to the project to import
4. Click "Add"
5. App crashes

**Expected behavior**
That the app either finds the correct `project.godot` file by itself or responds with a warning instead of crashing.

**Screenshots**
No screenshots, but a log:
```
Unhandled Exception:
System.UnauthorizedAccessException: Access to the path '/home/mai/apps/godot_re/rpginabox' is denied.
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x000e0] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions)
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize) [0x00055] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Boolean detectEncodingFromByteOrderMarks) [0x00007] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamReader..ctor(string)
  at ProjectConfig.Load (System.String fname) [0x00040] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ProjectFile.ReadFromFile (System.String filePath) [0x0000d] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ImportProject.OnAddBtnPressed () [0x000a5] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 

Unhandled Exception:
System.UnauthorizedAccessException: Access to the path '/home/mai/apps/godot_re/rpginabox' is denied.
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.Boolean anonymous, System.IO.FileOptions options) [0x000e0] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.FileStream..ctor (System.String path, System.IO.FileMode mode, System.IO.FileAccess access, System.IO.FileShare share, System.Int32 bufferSize, System.IO.FileOptions options) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.FileStream..ctor(string,System.IO.FileMode,System.IO.FileAccess,System.IO.FileShare,int,System.IO.FileOptions)
  at System.IO.StreamReader..ctor (System.String path, System.Text.Encoding encoding, System.Boolean detectEncodingFromByteOrderMarks, System.Int32 bufferSize) [0x00055] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path, System.Boolean detectEncodingFromByteOrderMarks) [0x00007] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at System.IO.StreamReader..ctor (System.String path) [0x00000] in <02071a39c8a34187bdd1f58e4be38764>:0 
  at (wrapper remoting-invoke-with-check) System.IO.StreamReader..ctor(string)
  at ProjectConfig.Load (System.String fname) [0x00040] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ProjectFile.ReadFromFile (System.String filePath) [0x0000d] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
  at ImportProject.OnAddBtnPressed () [0x000a5] in <f7b093cfd2f94b2ca92bd10d8f1454ab>:0 
```

**Complete the Following:**
 - OS: Linux
 - Version 0.2.1

**Comment by eumario** [comment]
Need to ensure that the path being returned appends project.godot to the end of the path when importing from a folder.  This will be fixed in 0.3.0

**Comment by eumario** [comment]
Fixed with commit [1ff936d](https://github.com/eumario/godot-manager/commit/1ff936d058aaf748325e71bcb183a85cd36c43c6).
